auslib/web/admin/views/problem.py

```diff
@@ -0,0 +1,44 @@
+import json
+from flask import Response
+
+
+def problem(status, title, detail, type=None, instance=None, headers=None, ext=None):
```

Comment:
The response error is like this 

> using connexion.problem results in TypeError: 'ConnexionResponse' object is not callable .

I couldn't debug it since the error was sporadic. It worked for some APIs but surprisingly not for all everytime. I tried to debug it but couldn't in the end. So that's why I decided to re-write the same method. This way it solves my issue for now and in future if we want to re-use connexion.problem all we have to do is simple one line change in the import statement from 'auslib.admin.views.problem" to "connexion.problem" .